src/Bicep.LangServer.IntegrationTests/CompletionTests.cs

```diff
@@ -2331,5 +2331,167 @@ private static BicepFile ApplyCompletion(BicepFile bicepFile, CompletionItem com
 
             return SourceFileFactory.CreateBicepFile(bicepFile.FileUri, replaced);
         }
+
+
+        [DataTestMethod]
+        [DataRow("loadTextContent")]
+        [DataRow("loadFileAsBase64")]
+        [DataRow("loadJsonContent", true)]
+        public async Task LoadFunctionsPathArgument_returnsFilesInCompletions(string functionName, bool jsonOnTop = false)
```

Comment:
Please could you also add some tests under `DefinitionTests.cs` for the "go to source" part of this PR?